spec/lib/terraform/runner_spec.rb

```diff
@@ -352,4 +352,60 @@ def verify_req(req)
       end
     end
   end
+
+  context '.parse_template_variables hello-world' do
+    describe '.parse_template_variables input/output vars' do
+      template_variables_stub = nil
+
+      def verify_req(req)
+        body = JSON.parse(req.body)
+        expect(body).to(have_key('templateZipFile'))
+      end
+
+      before do
+        ENV["TERRAFORM_RUNNER_URL"] = "https://1.2.3.4:7000"
+
+        @hello_world_variables_response = JSON.parse(File.read(File.join(__dir__, "runner/data/responses/hello-world-variables-success.json")))
```

Comment:
Why is this an instance var?  It looks like it is only used here https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/75/files#diff-3368830fa87ae1d0043c2385e9f9e0276e1e352e87c3fced3ca229203f1f6327R374